modules/core/shared/src/main/scala/lucuma/core/model/sequence/Dataset.scala

```diff
@@ -67,7 +67,7 @@ object Dataset {
       Option.when(localDate.getYear >= 0 && localDate.getYear <= 9999)(Filename(site, localDate, index))
 
     private val DateFormatter: DateTimeFormatter =
-      DateTimeFormatter.ofPattern("yyyyMMdd")
+      DateTimeFormatter.ofPattern("uuuuMMdd")
```

Comment:
Uhh interesting, is this working on the js side of things?

Reply:
Yes, now passes both JVM and JS.